Static SSR pages in a globally-interactive app (#32599)
dotnet/AspNetCore.Docs: aspnetcore/blazor/security/server/index.md

  <HeadOutlet @rendermode="new InteractiveServerRenderMode(prerender: false)" />
  ```

>   You can also selectively disable prerendering with fine control of the render mode applied to the `Routes` component instance. For more information, see <xref:blazor/components/render-modes#static-ssr-pages-in-a-globally-interactive-app>.

Reviewer: Hmm, I'm not sure what this sentence is trying to say. Can we clarify this point? Is this just trying to point out that you can selectively change the render mode of the `Routes` component?

```suggestion
  You can also selectively control the render mode applied to the `Routes` component instance. For example, see <xref:blazor/components/render-modes#static-ssr-pages-in-a-globally-interactive-app>.
```

Author: Addressed on https://github.com/dotnet/AspNetCore.Docs/pull/32613.
